### PR TITLE
433MHz Support

### PR DESCRIPTION
**Pull request by RaHoni**
This is the Pull request for #10

### CODE REVIEW

File: src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala [hunk 1]
```diff
@@ -41,23 +44,32 @@ case class mhzSwitch(setupID: String, _keepStatus: Boolean, private var _systemC
...
    */
-  override def off(): Unit = queue.queue.enqueue(mhzCommand(_systemCode, _unitCode, command = false))
+  override def off(): Unit = sendMhz(mhzCommand(_systemCode, _unitCode, command = false))
 
   /**
    * Generates a XML Node containing all Information to Regenerate the object
+   *
    * @return The Node Object
    */
   override def toXml: Node = <switch>
```
Comment (maximiliani) [inline src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: Please use <switch id ="{id}">. Otherwise editXML.scala wouldn't work correctly.

Comment (RaHoni) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: What did you mean with .?

Comment (RaHoni) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: @maximiliani

Comment (RaHoni) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: @maximiliani

Comment (RaHoni) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: @maximiliani

Comment (maximiliani) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: In toXml you used <switch>! This causes problems in the editXML.scala! Please use <switch id="{id}">! Thanks

Comment (maximiliani) [reply src/main/scala/PrivateHome/Devices/MHz/mhzSwitch.scala]: In toXml you used //<switch//>! This causes problems in the editXML.scala! Please use //<switch id //= //"id//"//> ! Thanks

File: src/main/scala/PrivateHome/Devices/MHz/GpioPinListener.scala [hunk 2]
```diff
@@ -12,16 +12,28 @@ class GpioPinListener extends GpioPinListenerDigital{
   private var lastTime:Long = 0
   private var repeatCount = 0
 
+
+  /**
+   * Gives the absolute difference
+   * @param a First number
+   * @param b Second number
+   * @return The result of the Calculation
```
Comment (maximiliani) [inline src/main/scala/PrivateHome/Devices/MHz/GpioPinListener.scala]: :)

**Pull request merged**